dashboard/src/actions/repos.ts

```diff
@@ -139,7 +139,12 @@ export const addRepo = (
     } = getState();
     try {
       dispatch(addOrUpdateRepo());
-      const namespaceScoped = !isGlobalNamespace(namespace, request.plugin?.name, config);
+      // TODO(agamez): improve the global/namespaced repos UX
+      // to avoid hacks like this (flux repos are always global)
+      // https://github.com/vmware-tanzu/kubeapps/issues/2995
+      const namespaceScoped =
+        !isGlobalNamespace(namespace, request.plugin?.name, config) &&
+        request.plugin?.name !== PluginNames.PACKAGES_FLUX;
```

Comment:
`isGlobalNamespace` has a switch per plugin.
Why not just returning `true` for the Flux case inside that function?

Reply:
Yep, good point. I guess it makes sense for the `isGlobalNamespace` function to always return `true` for flux packages. I'll do it.

Reply:
Done at https://github.com/vmware-tanzu/kubeapps/pull/5218/commits/3f8b65c8551ec387bcfdd9ee9fe7386d2813fe28